src/main/java/org/littleshoot/proxy/impl/ClientToProxyConnection.java

```diff
@@ -355,34 +355,26 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
             final HttpRequest httpRequest = (HttpRequest) msg;
 
             if (ProxyUtils.isChunked(httpRequest)) {
-                process(ctx, httpRequest);
+                process(ctx, httpRequest, true);
```

Comment:
we need it so the message is not released when we use `chunks` (when we do not use custom executor)